### PR TITLE
Update client-sdk-porting-guide.md

### PR DESCRIPTION
**Pull request by sharoncornelius**

### CODE REVIEW

File: docs/client-sdk/client-sdk-porting-guide.md [hunk 1]
```diff
@@ -1073,22 +1040,21 @@ int32_t fdo_con_recv_msg_header(fdo_con_handle handle,
...
 
-!!! note
-    This function may require a minimal change in implementation for porting to custom platform, as the reference implementation relies on Linux* libraries to receive data from the server.
+***NOTE:*** This function may require a minimal change in implementation for porting to custom platform, as the reference implementation relies on Linux* libraries to receive data from the server.
 
 *Parameters*
 
 `handle:` Connection handle. Please refer to `fdo_con_connect()`
 
 `protocol_version:` Incoming protocol version. Client SDK supports 113
```
Comment (DukeDavis12) [inline docs/client-sdk/client-sdk-porting-guide.md]: Whether we support 113 ?

**Review by KiranSukhavasi — CHANGES_REQUESTED** [review]
protocol version should be fdo 100 rather 113

File: docs/client-sdk/client-sdk-porting-guide.md (hunk 1 above)
Comment (prateekcr) [reply docs/client-sdk/client-sdk-porting-guide.md]: Should be 100.
Please update.

File: docs/client-sdk/client-sdk-porting-guide.md [hunk 2]
```diff
@@ -1073,22 +1040,21 @@ int32_t fdo_con_recv_msg_header(fdo_con_handle handle,
...
 
-`message_type:` set to FDO_TYPE_ERROR in case of error. Please refer FDO_TYPE_ERROR
+`message_type:` Set to FDO_TYPE_ERROR in case of error. Please refer FDO_TYPE_ERROR
 
 `msglen:` Length of incoming message body
 
-`ssl:` valid SSL context in case SSL is enabled. Please refer s`do_con_connect()`
+`ssl:` Valid SSL context in case SSL is enabled. Please refer to `do_con_connect()`
```
Comment (prateekcr) [inline docs/client-sdk/client-sdk-porting-guide.md]: Correction-
"Please refer to `fdo_con_connect()`"

Please update.

**Review by prateekcr — CHANGES_REQUESTED** [review]

**Review by DukeDavis12 — APPROVED** [review]

**Comment by trbehera** [comment]
Updated in #30

**Pull request closed without merging**